packages/fxa-auth-server/lib/routes/subscriptions.js

```diff
@@ -461,8 +539,7 @@ class DirectStripeRoutes {
       subscriptionId: sub.id,
       isActive,
       productId: sub.productId,
-      productCapabilities:
```

Comment:
And here's where we use the new `getProductCapabilities` method, since the source for the info in config is going away.